plinth/modules/first_boot/templates/firstboot_state10.html

```diff
@@ -27,28 +27,15 @@
 
     <h2>Setup Complete!</h2>
 
-    <p>FreedomBox setup is now complete.  However, you should check
-      the network setup and modify it if necessary.  Do not forget to
-      change the default Wi-Fi passwords.</p>
+    <p>The {{ cfg.box_name }} setup is now complete.  However, you should
+    <a href="{% url 'networks:index' %}">check the network setup</a> and
+    modify it if necessary.  Do not forget to change the default Wi-Fi
+    passwords!</p>
 
-    <p>To make your FreedomBox functional, you need some applications.
-      Applications will be installed the first time you access them.</p>
-
-    <h3>Network Configuration</h3>
+    <h3>Current Network Configuration</h3>
 
     {% include "connections_diagram.html" %}
 
-    <div class="row">
-      <div class="col-xs-6 text-center">
-        <a class="btn btn-lg btn-primary" href="{% url 'networks:index' %}">
-          Go to Networks</a>
-      </div>
-
-      <div class="col-xs-6 text-center">
-        <a class="btn btn-lg btn-primary" href="{% url 'apps:index' %}">
-          Go to Apps</a>
-      </div>
-    </div>
   </div>
```

Comment:
I think giving a specific direction of where to go to from here is a good thing.   The two buttons are helping with that.  I agree with your comment that the two buttons are looking as if they are related to 'Internal' and 'External' network options.  How about we move the "Apps" button just above the "Current Network Configuration" heading?  That way, the two button ugly layout is solved and we are also showing directions on where to go from here.

I believe, that the use of the definite article "The" before FreedomBox here is not correct.

De-emphasizing the links to go to network configuration is fine normally except that users will typically have to connect their devices to FreedomBox next and they need to understand where to connect them.  The networking button then also needs to be emphasized for the changes that may be necessary.

Changing the title to say "Current Network Configuration" is a nice improvement.

If it is okay with you, I will make the necessary changes and commit.